plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAChecker.java

```diff
@@ -71,6 +78,46 @@ public Boolean checkingHeartBeat() {
             }
         }
 
+        for (RbdStoragePool rbdpools : rbdStoragePools) {
+            hostAndPools = String.format("host IP [%s] in pools [%s]", hostIp, rbdStoragePools.stream().map(pool -> pool._monHost).collect(Collectors.joining(", ")));
+            s_logger.debug(String.format("Checking heart beat with KVMHAChecker for %s", hostAndPools));
+
+            ProcessBuilder processBuilder = new ProcessBuilder();
+            processBuilder.command().add("python3");
+            processBuilder.command().add(s_heartBeatPathRbd);
+            processBuilder.command().add("-i");
+            processBuilder.command().add(rbdpools._poolSourceHost);
+            processBuilder.command().add("-p");
+            processBuilder.command().add(rbdpools._poolMountSourcePath);
+            processBuilder.command().add("-n");
+            processBuilder.command().add(rbdpools._poolAuthUserName);
+            processBuilder.command().add("-s");
+            processBuilder.command().add(rbdpools._poolAuthSecret);
+            processBuilder.command().add("-v");
+            processBuilder.command().add(hostIp);
+            processBuilder.command().add("-r");
+            processBuilder.command().add("r");
+            Process process = null;
+            String parsedLine = "";
+            try {
+                process = processBuilder.start();
+                BufferedReader bfr = new BufferedReader(new InputStreamReader(process.getInputStream()));
+                parsedLine = bfr.readLine();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
```

Comment:
We should use the class `com.cloud.utils.script.Script` here (same as it is being doing to NFS), as it already does all the command handling.